templates/target.js

```diff
@@ -0,0 +1,44 @@
+export default ({markup, script, page, props}) => {
+  return `
+    <!DOCTYPE html>
+    <html>
+      <head>
+    	<title>BookBrainz – The Open Book Database</title>
```

Comment:
Interpolate a [ternary expression](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator) so that the contents of the `<title>` tags will become `YOUR_TITLE – The Open Book Database` if the `title` variable exists. Otherwise, the title will remain `BookBrainz – The Open Book Database`